### ISSUE TITLE
[web]: i-beam cursor not activated on hovering over a text field

### ISSUE THREAD

**Issue by boxabirds** [flutter/flutter#57142]
Flutter (Channel beta, v1.17.0, on Mac OS X 10.14.6 18G4032, locale en-GB)

Chrome's behaviour for a text field is that the cursor turns to an i-beam when the mouse moves over a field. Flutter web doesn't change the cursor until a click and it feels alien and disorienting. 

[Demo](https://youtu.be/WCJ99OyzJBk)

**Comment by kunit1** [comment]
Potentially related [PR](https://github.com/flutter/engine/pull/17718)

**Comment by TahaTesser** [comment]
This [issue](https://github.com/flutter/flutter/issues/31952) might consider this
Closing as duplicate of https://github.com/flutter/flutter/issues/31952

**Comment by dkwingsmt** [comment]
This issue will be solved once https://github.com/flutter/flutter/pull/57628 is merged (and published).
